NomaiVR/UI/Dialogue.cs

```diff
@@ -9,6 +9,11 @@ internal class Dialogue : NomaiVRModule<Dialogue.Behaviour, Dialogue.Behaviour.P
         protected override bool IsPersistent => false;
         protected override OWScene[] Scenes => PlayableScenes;
 
+        /// <summary>
+        /// Player is currently in a conversation
+        /// </summary>
+        public static bool IsActive { get; private set; }
```

Comment:
I'd rather call this `IsInConversation`, which would mean you don't need the comment anymore.

Also, I can't check right now, but I feel like maybe Outer Wild's `PlayerData` static class already has something like an "in conversation" flag? Might be worth checking.

Reply:
I actually had `IsInConversation` at 2 points, as i oscillated between different names that all seemed to come short 😅 

As the property is on `Dialogue`, i find it strange to read `Dialogue.IsInConversation`, it feels like it should really be `Player.IsInConversation` in that case. For the same reason i discarded `Dialogue.ConversationInProgress` and similar things, because it just did not parse quite nicely when reading it. I am also not very happy with `Dialogue.IsActive` either, so definitely open to changing it around a bit more.

I will look into `PlayerData`, this might be the most elegant solution if it has a flag for that.

Reply:
I just checked it, it's actually `PlayerState.InConversation()`. So yeah, no need to changes the patches in this case, awesome 😁

Reply:
👍 Will adjust it accordingly